waspc/cli/src/Wasp/Cli/Command/CreateNewProject.hs

```diff
@@ -122,6 +124,9 @@ createNewProject' (ProjectInfo projectName appName) = do
     mainWaspFileContent =
       unlines
         [ "app %s {" `printf` appName,
+          "  wasp: {",
+          "    version: \"^%s\"" `printf` Data.Version.showVersion Paths_waspc.version,
```

Comment:
@Martinsos and @NeoLight1010, what do you guys think, should we abstract this (Wasp's current version) under a module? It's pretty low level and we use it in several places.

If we extract it into a module, the usages becomes a bit cleaner:
```haskell
import Environment (waspVersion) -- I made up a random name

-- And then we just refer to 'waspVersion'

Reply:
I think it's a good idea :D using `Paths_waspc` is kind of unintuitive for a Haskell beginner like me. I think `Environment` is a good name for the module.

Reply:
Sounds interesting! It would be nice because we can endow it with extra information -> we know it is semantic version type of version. So we can have it have that type.
We can put it under smth like `Wasp.Version`, instead of using generic name like `Environment`.